.gitbook/0-about-panacea/1-data-exchange-protocol/1-DEP-problems-it-solves.md

```diff
@@ -0,0 +1,53 @@
+# DEP and the problems it solves
+
+DEP seeks to address the problems of decentralized data delivery that allows for data validation while ensuring data privacy.
+
+DEP aims to stay true to the five principles below:
+- [Data privacy](#data-privacy)
+- [Data validation](#data-validation)
+- [Incentive](#incentive)
+- [Decentralization](#decentralization)
+- [Scalability](#scalability)
+
+## Data privacy
+
+Since DEP is targeting sensitive data, the privacy of the data must be ensured throughout all data transfers. 
+No data must be exposed to anyone except promised data consumers during the data delivery.
+
+DEP uses data encryption in the whole data delivery process to ensure the privacy of sensitive data.
+[ECIES hybrid encryption](https://cryptobook.nakov.com/asymmetric-key-ciphers/ecies-public-key-encryption) is used to ensure data privacy during the entire data delivery process.
+
+## Data validation
+
+As data consumers, it is crucial to validate that the data they receive is correct.
+The data must be truthful, untampered, and conform to the requirements the consumer specified.
+
+DEP provides a way to verify data by utilizing oracle running in an secure enclave environment and DID-based verifiable presentation.
+
+## Incentive
+
+By providing data, the data provider can be guaranteed an incentive paid by the data consumer. 
+In this case, data delivery and payment must be processed atomically and simultaneously.
+
+DEP ensures this through Panacea blockchain transactions and data encryption.
+
+## Decentralization
+
+DEP aims to be a decentralized protocol with no centralized mediator for data validation and delivery.
```

Comment:
Does we need to say `with no centralized mediator`?

Reply:
could you please elaborate or suggest other options? i also think this sentence can be better but also it's okay as it is for now.

Reply:
I just feel that the `decentralized protocol with no centralized mediator` little awkward. But for now, I think it is okay to go.

Reply:
Man, if you feel it's awkward, please give us your suggestion. If not, the PR author doesn't understand what you mean.

Reply:
I guess you feel it's awkward to use `mediator` together with `centralized`

Reply:
Ah, I mean that it's awkward to use `decentralized protocol` together with `with no centralized mediator`. Because everybody knows that decentralized means that already didn't have any mediator.
So, I want to suggest like this:
`DEP aims to be a decentralized protocol for data validation and delivery.`
Sorry for the not detailed comment.

Reply:
I think it's more of a statement of emphasis. It's not partially decentralized, it's fully decentralized in the validation and delivery process.

Let's think of some other ways to express it.

Reply:
Ah, if you are using the expression for emphasis, it is not awkward.

Reply:
i agree with opinions from both of you. 
i'm okay with the emphasis that MS used. IMHO, i'm not trying to touch expression details at this time unless the meaning of the sentence is unclear.